index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <div class="menu-bar">
+      <div class="menu">
+        <div id="bar1" class="bar"></div>
+        <div id="bar2" class="bar"></div>
+        <div id="bar3" class="bar"></div>
+      </div>
+    </div>
+    <!-----------Header section------------->
+    <header>
+      <div class="main-header__brand">
+        <a href="index.html">
+          <img src="images/logo.png" alt="Logo" />
+        </a>
+      </div>
+    </header>
+
+    <section class="main-section">
+      <h1 class="main-section__introduction">I'm Mariana Glad to see you</h1>
+      <h6 class="main-section__text">
```

Comment:
- [ ] I think it will be better if you wrap your headline text description in a `<p>` tag which will make your website semantic which will in turn increase its accessibility 👍🏽